### PR TITLE
[Excel] (Custom Functions) Rename topics

### PR DESCRIPTION
**Pull request by mscharlock**
Took out the "Custom functions" in front of every name in the TOC because that was silly looking

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[7e79110](https://github.com/OfficeDev/office-js-docs-pr/pull/907/commits/7e79110eff4137ff427c9871f539be0f06bf780f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-cfsrename/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C18%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904182319544593-907%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=FLjQnl4ptP6ILKefESBAaC9aTnDB1VqUlfdxEK11pX0%3D&st=2019-04-18T23%3A16%3A01Z&se=2019-05-19T23%3A21%3A01Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
Docs Build status updates of commit _[159eb7f](https://github.com/OfficeDev/office-js-docs-pr/pull/907/commits/159eb7fcd284840d9c5e00932a6d99cacfb194cb)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/toc.yml](https://github.com/OfficeDev/office-js-docs-pr/blob/michelle-cfsrename/docs/toc.yml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C4%5C23%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201904231652183304-907%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=vq8YAWzQX9x%2BWf79Tx15CoqJrUMYxFD0D8udtuprOKE%3D&st=2019-04-23T16%3A48%3A29Z&se=2019-05-24T16%3A53%3A29Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/toc.yml [hunk 1]
```diff
@@ -269,16 +269,16 @@
...
           displayName: Excel
-        - name: Web requests
+        - name: Making web requests
           href: excel/custom-functions-web-reqs.md
           displayName: Excel
         - name: Runtime
           href: excel/custom-functions-runtime.md
           displayName: Excel
-        - name: Metadata
+        - name: JSON Metadata
```
Comment (AlexJerabek) [inline docs/toc.yml]: Lowercase "metadata"

File: docs/toc.yml [hunk 2]
```diff
@@ -269,16 +269,16 @@
...
           href: excel/custom-functions-json.md
           displayName: Excel
-        - name: Custom function metadata
+        - name: Autogenerate metadata
```
Comment (AlexJerabek) [inline docs/toc.yml]: Considering this article discusses manual steps the customer needs to do, maybe this title should be "Metadata generation"?

**Comment by lindalu-MSFT** [comment]
@mscharlock Ugh, So sorry for the overlap here, but we no longer need this PR! Linda Caputo already merged the same changes to Master, and I'll push those live today.

**Comment by AlexJerabek** [comment]
@lindalu-MSFT @mscharlock Gotcha. Well, please consider this feedback against the current version :-)

**Comment by lindalu-MSFT** [comment]
@AlexJerabek @mscharlock Good point, Alex. I'll publish tonight, then for sure please review and make whatever suggestions were overlooked. THANKS

**Pull request closed without merging**